src/index.js

```diff
@@ -189,29 +218,32 @@ async function replicate({ seq }) {
       return_docs: false, // eslint-disable-line camelcase
     });
 
-    const q = cargo((docs, done) => {
-      saveDocs({ docs, index: mainIndex })
-        .then(() => infoChange(docs[docs.length - 1].seq, 1, '🐌'))
-        .then(() =>
-          stateManager.save({
-            seq: docs[docs.length - 1].seq,
-          })
-        )
-        .then(() => done())
-        .catch(done);
+    const q = cargo(async docs => {
```

Comment:
did you update async in this PR? I think we are a major version behind, which works a lot better with promises

Reply:
done